pkg/uconn/analyzer.go

```diff
@@ -140,15 +144,20 @@ func (a *analyzer) hostMaxDurQuery(maxDur float64, localIP string, externalIP st
 	// create query
 	query := bson.M{}
 
+	//TODO: Why do we update old chunks with new max durations??? (https://github.com/activecm/rita/pull/512/files) -LL
```

Comment:
This query updates max duration entries for a given ip pair, but it updates the entries within all of the available chunks. TBH I'm not sure how this query really works or where the consuming view logic is at.